src/grandorgue/GOOrganController.cpp

```diff
@@ -137,6 +145,8 @@ GOOrganController::~GOOrganController() {
   m_tremulants.clear();
   m_ranks.clear();
   m_VirtualCouplers.Cleanup();
+  m_timer->Cleanup();
+  m_bitmaps->Cleanup();
```

Comment:
```suggestion
  if (m_timer)
    delete m_timer;
  if (m_bitmaps)
    delete m_bitmaps;
```

Cleanup() should be called from the destructors instead of there

Reply:
That was the implementation a little above. That's why I mimicked.